Lil-Peep/src/styles/modules/related-products.scss

```diff
@@ -0,0 +1,72 @@
+.related-product-container {
+  background-color: $product-card-category;
+  padding-left: 30px;
+  padding-right: 30px;
+
+  .collection-container {
+    background-color: $product-card-category;
+
+    .product-image {
+      justify-content: space-between;
+
+      a {
+        &:nth-of-type(odd) {
+          @media #{$breakpoint-tablet} {
+            display: none;
+          }
+        }
+      }
+
+      @media #{$breakpoint-tablet} {
+        justify-content: center;
+      }
+    }
+
+    .product-card-category {
+
+      &:last-child {
+
+        @media #{$last-child-breakpoint} {
+          display: none;
+        }
+      }
+
+      &:nth-last-child(2) {
+
+        @media #{$penultimate-child-breakpoint} {
+          display: none;
+        }
+      }
+
+      @media #{$breakpoint-mobile} {
+        max-width: none;
+      }
+    }
+
+    @media #{$breakpoint-tablet} {
+      background-color: $background-color;
+    }
+  }
+
+  @media #{$breakpoint-tablet} {
+    background-color: $background-color;
+  }
+}
+
+.related-product-title {
+  color: $title-font-gray;
+  font-family: $lato-font;
+  font-size: 25px;
+  font-weight: 100;
+  padding-top: 20px;
+  text-align: center;
+
+  @media #{$breakpoint-tablet} {
+    margin-left: 10px;
+    text-align: left;
+
+    &:after {
+      content: ":";
```

Comment:
Is this necessary? 🤔

Reply:
yep, look at this:

<img width="122" alt="captura de pantalla 2018-03-29 a la s 11 47 23" src="https://user-images.githubusercontent.com/23444805/38104639-06c2a866-3347-11e8-91af-af364f2c9a32.png">
<img width="156" alt="captura de pantalla 2018-03-29 a la s 11 47 32" src="https://user-images.githubusercontent.com/23444805/38104640-06d743fc-3347-11e8-90d9-1909ac8364e9.png">

Reply:
differences between mobile and desktop versions 🤔

Reply:
Oh, I see, looks like this was a design error, but yeah, you accomplished the design, that's nice :)